pybossa/view/admin.py

```diff
@@ -577,6 +579,20 @@ def management_dashboard():
                            task_chart=task_chart,
                            submission_chart=submission_chart)
 
+@blueprint.route('/usage_dashboard/')
+@login_required
+@admin_required
+def usage_dashboard():
+    str_months = str(request.args.get('months'))
+    if not str_months.isnumeric():
```

Comment:
Should the code check if 'months' exists in the request args? It might just return an empty string if it does not exist, but it's a good idea to confirm this.

Reply:
if 'months' does not exist, `str_months` will be None and `str_months.isnumeric()` will return false. See [test case](https://github.com/bloomberg/pybossa/blob/54f26a9409a910cb71c176dcafb94241e947bfde/test/test_admin.py#L1350)